sk8.sh

```diff
@@ -2574,8 +2574,7 @@ EOF
   fi
 
   cat <<EOF >/etc/default/kubelet
-KUBELET_OPTS="--allow-privileged \\
---client-ca-file='${TLS_CA_CRT}'${EXT_CLOUD_PROVIDER_OPTS} \\
+KUBELET_OPTS="--client-ca-file='${TLS_CA_CRT}'${EXT_CLOUD_PROVIDER_OPTS} \\
```

Comment:
Hi @yastij,

Any negative affect on a versions of Kubernetes built before [this commit](https://github.com/kubernetes/kubernetes/pull/77820)?

Reply:
this should be fine as it already defaults to true to supported version see kubernetes/kubernetes#63442